ipatests/ipa-run-tests

```diff
@@ -33,13 +33,16 @@ import sys
 import pytest
 
 import ipalib
-import ipatests
+
+try:
+    import ipatests
+    HERE = os.path.dirname(os.path.abspath(ipatests.__file__))
+except ImportError:
+    HERE = os.getcwd()
```

Comment:
this (and in `conftest.py`)... belong to a `WIP: Make tests work locally for me.` commit?

Should be cleaned up... :)

Reply:
Yeah, I had a few extra local patches I forgot to remove before last pushing.